src/Datadog.Trace.ClrProfiler.Native/util.cpp

```diff
@@ -66,10 +66,10 @@ WSTRING GetEnvironmentValue(const WSTRING &name) {
 #endif
 }
 
-std::vector<WSTRING> GetEnvironmentValues(const WSTRING &name,
-                                          const wchar_t delim) {
+std::vector<WSTRING> GetValues(const WSTRING &delimited_values,
```

Comment:
After this changes, we may want to rename this method to something like `SplitAndTrim()` .

Reply:
Good idea, implemented in 1e9f67187108f53bf9612eb17925e8106d5a93c5